### PR TITLE
Adding a new AHU

### PR DESCRIPTION
**Pull request by Ciaran-Oflannagain**
@kevin-hereworks

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 1]
```diff
@@ -17,6 +17,22 @@
...
 
 ### Single Zone Units ###
+AHU_SS_HTWHLSTC_SFVSC_EFVSC_EDM_SDM_RFC_SFC_FDPM2X_RTM:
+  guid: "e0627d5a-99cd-4edd-824b-636428427980"
+  description: "Single zone AHU, with supply and return fans, heatwheel and fan speed flowrate control"
+  is_canonical: temperature
+  implements:
+  - AHU
+  - SS
+  - HTWHLSTC
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:27]: If this has a heat wheel is it actually a DOAS unit (where the supply and exhaust air streams are separated from each other)?

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 2]
```diff
@@ -17,6 +17,22 @@
 ########################
 
 ### Single Zone Units ###
+AHU_SS_HTWHLSTC_SFVSC_EFVSC_EDM_SDM_RFC_SFC_FDPM2X_RTM:
+  guid: "e0627d5a-99cd-4edd-824b-636428427980"
+  description: "Single zone AHU, with supply and return fans, heatwheel and fan speed flowrate control"
+  is_canonical: temperature
+  implements:
+  - AHU
+  - SS
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:26]: What is this for? Typically we don't include SS if you have SFVSC

**Review by cstirdivant — DISMISSED** [review]

**Comment by cstirdivant** [comment]
Hi @Ciaran-Oflannagain I have a feeling this is actually a DOAS unit, where the supply and exhaust flow streams are isolated from each other and heat is transferred between the two via the heat wheel.

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml (hunk 1 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/AHU.yaml]: @cstirdivant - thanks have changed to DOAS unit, no need for SS abstract type. Ciaran

File: ontology/yaml/resources/HVAC/entity_types/DOAS.yaml [hunk 3]
```diff
@@ -16,6 +16,22 @@
...
+  implements:
+  - DOAS
+  - HTWHLSTC
+  - SFVSC
+  - EFVSC
+  - EDM
+  - SDM
+  - RFC
+  - SFC
+  - FDPM2X
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/DOAS.yaml:32]: Do you have any filter alarms on the device?

File: ontology/yaml/resources/HVAC/entity_types/DOAS.yaml [hunk 4]
```diff
@@ -16,6 +16,22 @@
 ### Canonical Types  ###
 ########################
 
+DOAS_HTWHLSTC_SFVSC_EFVSC_EDM_SDM_RFC_SFC_FDPM2X_RTM:
+  guid: "e0627d5a-99cd-4edd-824b-636428427980"
+  description: "DOAS with supply and return fans, heatwheel and fan speed flowrate control"
+  is_canonical: temperature
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/DOAS.yaml:22]: typo here, should be "is_canonical: true"

File: ontology/yaml/resources/HVAC/entity_types/DOAS.yaml [hunk 5]
```diff
@@ -16,6 +16,22 @@
 ### Canonical Types  ###
 ########################
 
+DOAS_HTWHLSTC_SFVSC_EFVSC_EDM_SDM_RFC_SFC_FDPM2X_RTM:
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/DOAS.yaml:19]: Please confim that there is no heating or cooling performed outsie of heat recovery

**Review by shambergoldstein — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/DOAS.yaml (hunk 5 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/DOAS.yaml:19]: @shambergoldstein - Heating and cooling only takes place in heatwheel within heat recovery

File: ontology/yaml/resources/HVAC/entity_types/DOAS.yaml (hunk 3 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/DOAS.yaml:32]: No filter alarms for this just the filter differential pressure sensors. Thanks

**Review by shambergoldstein — APPROVED** [review]

**Pull request merged**